src/components/dashboard/Claim.js

```diff
@@ -117,20 +122,25 @@ const Claim = props => {
   }
 
   const init = async () => {
-    //hack to make unit test pass, activityindicator in claim button cuasing
+    // hack to make unit test pass, activityindicator in claim button cuasing
     if (process.env.NODE_ENV !== 'test') {
```

Comment:
is this for **unit** tests or for cypress ? if it's for cypress it should be updated to isE2ERunning imported from utils/platform

Reply:
I checked it and found that it is for unit tests. Its solves 2 tests